recipes/sequence-model/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sequence-model" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sequence-dev/sequence/archive/v{{ version }}.tar.gz
+  sha256: 30e164a47b4110925bd74153b60a47e5ca900655ccfdf5f119049e79e997f0b1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - sequence=sequence.cli:sequence
+
+
+requirements:
+  host:
+    - python >=3.8
```

Comment:
Upstream says 3.6...any reason you're being more restricted?

Reply:
@dopplershift Nope, no good reason. I've reduced the requirement to *>=3.6*.